driver-mct/cime_config/buildnml

```diff
@@ -74,16 +74,20 @@ def _create_drv_namelists(case, infile, confdir, nmlgen, files):
     # ---------------------------------------------------
     if case.get_value('COMP_WAV') == 'ww3':
         config['WAVSPEC'] = case.get_value('WAV_SPEC')
+        config['WAV_ATM_COUP'] = 'twoway' if case.get_value('COMP_ATM') == 'eam' else 'oneway'
```

Comment:
does it make sense to include 'eamxx' as an option?  Or is that better left for the future.

Reply:
great question ... I'm open to building in wave- EAMxx coupling capabilities.

Reply:
go ahead.  Is this the only mod?

Reply:
My vote is to add eamxx at a later time.